framework/src/modules/poa/module.ts

```diff
@@ -331,5 +323,14 @@ export class PoAModule extends BaseModule {
 				throw new Error('Invalid validator key found in poa genesis asset validators.');
 			}
 		}
+
+		// Pass the required information to the Validators module.
+		await this._validatorsMethod.setValidatorsParams(
+			methodContext,
+			context,
+			currentRoundSnapshot.threshold,
+			currentRoundSnapshot.threshold,
+			currentRoundSnapshot.validators.map(v => ({ address: v.address, bftWeight: v.weight })),
+		);
```

Comment:
not sure why it was moved here for this PR we should not touch the source code, if its a bug then create an issue and separate PR

Reply:
Created
https://github.com/LiskHQ/lisk-sdk/issues/8670